oauth2_http/java/com/google/auth/oauth2/ImpersonatedCredentials.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2018, Google Inc. All rights reserved.
+ * Copyright 2019, Google Inc. All rights reserved.
```

Comment:
This one should stay at 2018 - we keep it as the year the file was first introduced.